components/Notifications.tsx

```diff
@@ -7,9 +7,12 @@ import {
 } from '@heroicons/react/24/outline';
 import { XMarkIcon } from '@heroicons/react/20/solid';
 import { useNotifications } from '@/contexts/NotificationsContext';
+import { useNetwork } from 'wagmi';
 
 export default function Notifications() {
   const { notifications, dismiss } = useNotifications();
+  const { chain } = useNetwork();
+  const etherscanPrefix = chain?.id === 5 ? 'goerli' : `goerli-optimism`;
```

Comment:
this doesn't work for a few reasons:
- chain ids are set in the config file. if these chain ids change, the urls here would evaluate to goerli-optimism, which wouldn't be correct
- even if we stuck to goerli and goerli-optimism, `chain` here seems based on what network we're connected to. So if the user sent a tx on L1, and then changed networks to L2, the link would be for L2 -- but that would 404 because the tx is on L1.
- the approach of using an etherscan prefix doesn't quite work, because a chain like "base" for example would be basescan.org.

OK, let's brainstorm an approach or 2 that might solve some of the above frictions:

One idea is to add a property in the config for both l1 and l2 called `makeEtherscanLink`. the type of the property should be a method that contains 1 param (txHash). Then, here, we could do something like:

```ts
const {l1, l2} = useConfig();
...
return (
...
<a href={notification.chainId === l2.chain.id ? l2.makeEtherscanLink(notification.hash) : l1.makeEtherscanLink(notification.hash)}>View on Etherscan</a>
...
}
```

A different approach would be a helper function somewhere in a util file, that contains 

```ts
export const makeEtherscanLink = (chainId, hash) => `https://${etherscanUrl[chainId]]/tx/${hash}`
```

in the same file we could have:

```ts
const etherscanUrl = {
  1: "etherscan.io",
  5: "goerli.etherscan.io",
  420: "optimism-goerli.etherscan.io"
}
```

then in the notifications, we import the helper method and use it.

One issue here is that a notification does not currently include chain in its type. So let's make an issue saying that we need chainId in the notification type.

Reply:
Took the second approach of adding a helper function to `util`. But used `viem/chains` for the blockExplorer urls. [830eb53](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/55/commits/830eb5331948ce64e07ab1da444c8f2745288764)